src/Common/Services/Mail/MailBootstrap.cs

```diff
@@ -17,26 +17,12 @@ public static void AddMail(this WebApplicationBuilder builder)
             {
                 var section = builder.Configuration.GetSection("Mail:Providers");
 
-                if (!section.GetChildren().Any())
-                {
-                    o.Providers = new List<BaseMailProviderOptions>
-                    {
-                        new FileMailProviderOptions()
```

Comment:
Removing it might not be a good idea, we could define a new mail provider to log to the console using a `warning` or `info` log level for testing. Or output to Memorycache or some other in-memory option.

For self-hosting keeping the file is better by default.

Reply:
I think all you need to do to fix the API testing image is the following:

https://github.com/bitwarden/passwordless-server/pull/716

Reply:
It is ready, the tests are passing